@here/harp-map-controls/lib/MapControls.ts

```diff
@@ -802,7 +802,7 @@ export class MapControls extends THREE.EventDispatcher {
             return;
         }
 
-        event.preventDefault();
+        //event.preventDefault();
```

Comment:
is there some reason for this?

Reply:
Sorry, i forgot to push cleaned-up patch, back to question. Will do in a moment.

> is there some reason for this?

Actually, i didn't done very thorough research, but without `event.preventDefault` `mouseDown` events properly arrive, even when we drag mouse outside our iframe.

With `event.preventDefault()` we just don't get `mouseUp` that happens outside of our frame.

I don't know why, the only source is enigmatic blog post: https://taye.me/blog/tips/2015/11/16/mouse-drag-outside-iframe/

Reply:
So, summarizing it looks like  `window.top` listeners were needed for `iframe` scenario because we called `event.preventDefault`.
Now, without we can safely remove `window.top` event listeners and thus cross-origin issues.